packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -47,23 +39,30 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
     this._updateEventHandlers(nextProps);
   }
 
-  public componentWillUnmount(): void {
-    const { ignoreExternalFocusing } = this.props;
+  public componentDidUpdate(prevProps: IFocusTrapZoneProps) {
+    const prevForceFocusInsideTrap = prevProps.forceFocusInsideTrap !== undefined ? prevProps.forceFocusInsideTrap : true;
+    const newForceFocusInsideTrap = this.props.forceFocusInsideTrap !== undefined ? this.props.forceFocusInsideTrap : true;
+
+    if (!prevForceFocusInsideTrap && newForceFocusInsideTrap) {
+      // Transition from forceFocusInsideTrap disabled to enabled. Emulate what happens when a FocusTrapZone gets mounted
+      FocusTrapZone._focusStack.push(this);
```

Comment:
>FocusTrapZone._focusStack.push(this); [](start = 6, length = 37)

Seems a little odd that we are not pushing every time we are going to bringFocusIntoZone. Is that expected?